spark/spark-operator/base/kustomization.yaml

```diff
@@ -11,7 +11,7 @@ commonLabels:
 # creating patches.
 images:
 - name: gcr.io/spark-operator/spark-operator
-  newTag: v1beta2-1.0.0-2.4.4
+  newTag: v1beta2-1.1.0-2.4.5
 
 # Value of this field is prepended to the
```

Comment:
One problem I noticed is that this prefix causes the operator to be named "spark-operator-sparkoperator". Can we fix this?

Reply:
oh hmm. I don't know how to have a single field skip the prefix in Kustomize is that possible? Or are you suggesting a different prefix (e.g. `kf-spark`) and changing the operator name operator? I'm flex either way

Reply:
Is it ok if we save the operator renaming for another PR? I'd like to get this merged sooner rather than later given how often it gets out of sync.

Reply:
Sure.